app/src/main/res/values/strings.xml

```diff
@@ -13,20 +13,16 @@
     <string name="content_description_private_browsing_button">Enable private browsing</string>
     <!-- Content description (not visible, for screen readers etc.): "Private Browsing" menu button. -->
     <string name="content_description_disable_private_browsing_button">Disable private browsing</string>
-    <!-- Explanation for how sessions work -->
-    <string name="sessions_intro_description">Sessions help you return to the sites you visit often and those you\'ve
-        just discovered. Start browsing and they\'ll begin to appear here.
-    </string>
     <!-- Placeholder text shown in the search bar before a user enters text -->
     <string name="search_hint">Search or enter address</string>
 
     <!-- Private Browsing -->
     <!-- Title for private session option -->
-    <string name="private_browsing_title">You\'re in a private session</string>
+    <string name="private_browsing_title">You’re in a private session</string>
```

Comment:
I think this works. Apostrophe needs an escape character because it's considered a special character. Hopefully, right single quotation mark is not.

Reply:
It worked when we did it in Focus :)